src/validators.ts

```diff
@@ -73,7 +73,7 @@ export const isHotkeyMatchingKeyboardEvent = (e: KeyboardEvent, hotkey: Hotkey,
         return false
       }
     } else {
-      if (metaKey !== meta && ctrlKey !== meta && keyCode !== 'meta' && keyCode !== 'ctrl') {
+      if (metaKey !== meta && ctrlKey !== meta) {
```

Comment:
Fixes #909 - combinations such as `meta+k` end up in `keyCode` equalling `k` rather than the modifier